nodes/object/object_bounding_box.py

```diff
@@ -1,21 +1,31 @@
 import bpy
+from bpy.props import *
+from ... events import executionCodeChanged
 from mathutils import Vector
 from ... base_types.node import AnimationNode
 
 class ObjectBoundingBoxNode(bpy.types.Node, AnimationNode):
     bl_idname = "an_ObjectBoundingBoxNode"
     bl_label = "Object Bounding Box"
-
+    
+    useWorldSpace = BoolProperty(name = "Use World Space", default = True,
+        update = executionCodeChanged)
```

Comment:
we don't use custom execution code here. 
Using propertyChanged instead of executionCodeChanged is better.